MySelenium/src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,23 @@
+package tests;
+
+import driver.Driver;
+import driver.config.BaseDriverConfig;
+import driver.config.ChromeBuilder;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class BaseTest {
+    private static BaseDriverConfig baseDriverBuilder = new ChromeBuilder().withTimePageLoad(60).withTimeElementWait(40).build();
```

Comment:
implement factory pattern for this
at the end of the day it will be driverConfigFactory.getConfig(BrowserType.Chrome)